libs/langchain/langchain/llms/databricks.py

```diff
@@ -53,13 +53,7 @@ def set_api_url(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         return values
 
     def post(self, request: Any) -> Any:
-        # See https://docs.databricks.com/machine-learning/model-serving/score-model-serving-endpoints.html
-        wrapped_request = {"dataframe_records": [request]}
-        response = self.post_raw(wrapped_request)["predictions"]
-        # For a single-record query, the result is not a list.
-        if isinstance(response, list):
-            response = response[0]
-        return response
```

Comment:
@dbczumar This is a breaking change. If this isn't acceptable, we can define another client and add a new constructor argument to use it.

Reply:
let me know if you find a better idea.

Reply:
Yes, my idea is to fetch information about the endpoint via `get_endpoint()` when the instance of `llms.Databricks` is constructed. This info should contain a type. If the type is foundation model / external model, we can avoid wrapping. Else, we can wrap. We need to do this wrapping for non-foundation / non-external models.

Reply:
sg!